sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -803,6 +803,25 @@ class HiveDDLSuite
     }
   }
 
+  test("Insert overwrite directory should output correct schema") {
```

Comment:
Since this is a bug fix, can we have `SPARK-25313` prefix?

Reply:
Also added here?
https://github.com/apache/spark/blob/8e60b98239be63555644e013417cda7175baf984/sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala#L758
https://github.com/apache/spark/blob/8e60b98239be63555644e013417cda7175baf984/sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala#L782

Reply:
In this PR, let's handle this test case only.